Auth0Tests/AuthenticationErrorSpec.swift

```diff
@@ -364,6 +364,15 @@ class AuthenticationErrorSpec: QuickSpec {
                 expect(error.isRefreshTokenDeleted) == true
             }
 
+            it("should detect invalid refresh token") {
+                let values = [
+                    "error": "invalid_grant",
+                    "error_description": "Unknown or invalid refresh token."
+                ]
+                let error = AuthenticationError(info: values, statusCode: 403)
+                expect(error.isInvalidRefreshToken) == true
+            }
+
```

Comment:
Please update the "[should not match any custom error](https://github.com/auth0/Auth0.swift/blob/e7c227681733f816bdf5942261f8bbda950f338d/Auth0Tests/AuthenticationErrorSpec.swift#L475-L483)" and "[should not match any known error](https://github.com/auth0/Auth0.swift/blob/e7c227681733f816bdf5942261f8bbda950f338d/Auth0Tests/AuthenticationErrorSpec.swift#L536-L549)" test cases as well.

Reply:
Thanks for the quick review! Updated.